src/Microsoft.Win32.Registry/ref/Microsoft.Win32.Registry.csproj

```diff
@@ -9,5 +9,9 @@
   <ItemGroup>
     <Compile Include="Microsoft.Win32.Registry.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\System.Security.AccessControl\ref\System.Security.AccessControl.csproj" />
+    <ProjectReference Include="..\..\System.Security.Principal.Windows\ref\System.Security.Principal.Windows.csproj" />
```

Comment:
I'm not sure I like this dependency but we can deal with it for now. Can you confirm that it is only needed because of System.Security.Principal.IdentityReference?